### PR TITLE
Break out Gems into groups, add some helpful gems from EEC

### PR DESCRIPTION
**Pull request by alxjrvs**
The Work behind this PR was primarily created by @nialbima , I just cleaned it up and put it up for a PR!

### CODE REVIEW

**Review by nialbima — APPROVED** [review]
Sick, this looks good!

File: rails/7/gnarly.rb [hunk 1]
```diff
@@ -46,24 +46,48 @@ def create_gnarly_rails_app
...
+  gem_group :ci, :development, :test do
+    gem 'pronto'
+    gem 'pronto-rubocop', require: false
+    gem 'pronto-brakeman', require: false
+    gem 'pronto-erb_lint', require: false
+    gem 'gnar-style', require: false
+    gem 'rubocop-rspec', require: false
+  end
+
+  # I like groups.
```
Comment (alxjrvs) [inline rails/7/gnarly.rb]:
```suggestion
```

File: rails/7/files/.erb-lint.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      Style/FrozenStringLiteralComment:
+        Enabled: false
+      Style/TrailingCommaInArguments:
+        EnforcedStyleForMultiline: no_comma
+      Style/TrailingCommaInHashLiteral:
+        EnforcedStyleForMultiline: comma
+      Lint/UselessAssignment:
+        Enabled: true
+      Rails/OutputSafety:
+        Enabled: true
```
Comment (alxjrvs) [inline rails/7/files/.erb-lint.yml:42]: Question, @nialbima  - are these rules coming directly from EEC, and are there any that we might *generally* consider worthwhile that aren't used in EEC? (for instance, that variable naming one you mentioned in DM's - it makes sense for EEC to turn that off, but I think we can generally say that rule is worthwhile by default). 

Curious how much of this is sanitized from EEC

Comment (alxjrvs) [reply rails/7/files/.erb-lint.yml:42]: Additionally: We may need to do some work to integrate this fully into the Gnarly template. Lets circle back on this when we both have more cycles :D

**Pull request merged**